README.md.rej

```diff
@@ -0,0 +1,5 @@
+diff a/README.md b/README.md	(rejected hunks)
+@@ -1,2 +1,2 @@
+-[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat//14.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-vertical-rental-)
```

Comment:
@sbidoul this is weird that the runbot id is not put her and why it fails to merge. Maybe you miss something when running the script?

Reply:
The problem comes from the copier template that vendors maintainer tools as a git submodule that needs updating whenever a repo is added. https://github.com/OCA/oca-addons-repo-template/issues/87

In the meantime, we can just fix the runbot URL manually (vertical-rental runbot id is 298)

Reply:
Thanks for the clarifications, Stéphane. Let's see if my colleague knows about other strategy.